### ISSUE TITLE
Ambiguous class resolution WP_CLI\Inflector

### ISSUE THREAD

**Issue by szepeviktor** [wp-cli/wp-cli-bundle#41]
```
$ composer require --prefer-dist wp-cli/wp-cli-bundle --optimize-autoloader
Using version ^2.0 for wp-cli/wp-cli-bundle
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Nothing to install or update
Generating optimized autoload files
Warning: Ambiguous class resolution, "WP_CLI\Inflector" was found in both "/home/instahu/website/tmp/vendor/wp-cli/wp-cli/php/WP_CLI/Inflector.php" and "/home/instahu/website/tmp/vendor/wp-cli/scaffold-command/src/WP_CLI/Inflector.php", the first will be used.
```

It seem scaffold-command has a duplicated class.
Please advise.

**Comment by szepeviktor** [comment]
There was an intention to fix it but currently it seem to be broken.
https://github.com/wp-cli/scaffold-command/issues/164

**Comment by schlessera** [comment]
@szepeviktor This will be resolved for your case as soon as we release v2.1.0 of WP-CLI.

Also not that the ambiguous class resolution just points to two classes that are completely identical, so it will not even cause an error apart from the warning you get during autoload generation.

**Comment by szepeviktor** [comment]
> so it will not even cause an error

Thank you!

Errors grow out of tiny small warnings :)

**Comment by schlessera** [comment]
This should have been fixed with the release of v2.1.0.

**Comment by szepeviktor** [comment]
Thanks.
